ALUMNI.md

```diff
@@ -0,0 +1,12 @@
+# Alumni
+
+Thanks to the following folks who used to maintain Cosign!
+
+- @dlorenc
```

Comment:
I would drop the @ and sort alphabetical and mention that is sorted in the inital text

Reply:
I like the `@` to make it clear these are handles; I added links to GitHub profiles as well.

But sorting is a good idea, I've done that.